src/main/java/lotto/domain/LottoFactory.java

```diff
@@ -0,0 +1,27 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class LottoFactory {
+    private static final LottoNumber[] LOTTO_NUMBERS = {
+            new LottoNumber(1), new LottoNumber(2), new LottoNumber(3), new LottoNumber(4), new LottoNumber(5),
+            new LottoNumber(6), new LottoNumber(7), new LottoNumber(8), new LottoNumber(9), new LottoNumber(10),
+            new LottoNumber(11), new LottoNumber(12), new LottoNumber(13), new LottoNumber(14), new LottoNumber(15),
+            new LottoNumber(16), new LottoNumber(17), new LottoNumber(18), new LottoNumber(19), new LottoNumber(20),
+            new LottoNumber(21), new LottoNumber(22), new LottoNumber(23), new LottoNumber(24), new LottoNumber(25),
+            new LottoNumber(26), new LottoNumber(27), new LottoNumber(28), new LottoNumber(29), new LottoNumber(30),
+            new LottoNumber(31), new LottoNumber(32), new LottoNumber(33), new LottoNumber(34), new LottoNumber(35),
+            new LottoNumber(36), new LottoNumber(37), new LottoNumber(38), new LottoNumber(39), new LottoNumber(40),
+            new LottoNumber(41), new LottoNumber(42), new LottoNumber(43), new LottoNumber(44), new LottoNumber(45)
+    };
```

Comment:
미리 로또 넘버 배열을 만드는 건 잘 해주셨는데요:) 만약 로또 번호가 100개, 1000개가 된다면 어떻게 될까요?
차라리 Stream과 List를 이용해서 만드는게 나을 것 같습니다.
```suggestion
    private static final List<LottoNumber> LOTTO_NUMBERS = IntStream.rangeClosed(1, 45)
            .boxed()
            .map(LottoNumber::new)
            .collect(Collectors.toList());
```
여기서 1과 45라는 매직넘버도 상수화 해두면 좋겠죠:)

Reply:
이렇게 간소화 시킬 수 있군요! 감사합니다!ㅎㅎ